internal/exec/yaml_func_store.go

```diff
@@ -68,7 +89,7 @@ func processTagStore(atmosConfig schema.AtmosConfiguration, input string, curren
 	store := atmosConfig.Stores[retParams.storeName]
 
 	if store == nil {
-		u.LogErrorAndExit(fmt.Errorf("invalid Atmos Store YAML function execution:: %s\nstore '%s' not found", input, retParams.storeName))
+		log.Fatal(fmt.Errorf("%w: %s\nstore '%s' not found", ErrInvalidYamlFunc, input, retParams.storeName))
```

Comment:
Why are we using format specifiers and error format instead of structured log event?

Reply:
No, the the reason we added this lint warning is to raise the problem that we are not using the new logger methods. The point of the new logger methods is to use structured logging. Updating to use the new logger methods without using structured logging defeats the purpose.